task5/outputs.tf

```diff
@@ -0,0 +1,28 @@
+output "ecs_ami_id" {

```

Comment:
Please add relevant public IP address output. 
or null resource to check if container is started.